.github/workflows/dependabot.yml

```
@@ -0,0 +1,75 @@
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: gradle-${{ runner.os }}-${{ hashFiles('backend/**/*.gradle*', 'backend/settings.gradle*', 'backend/gradle.properties', 'backend/gradle/wrapper/gradle-wrapper.properties') }}
+          restore-keys: |
+            gradle-${{ runner.os }}-
+
+      - name: Refresh lockfiles
+        working-directory: backend/
+        run: ./gradlew dependencies --write-locks
+
+      - name: Save Gradle Cache
+        if: ${{ always() && inputs.save_gradle_cache }}
```

actionlint (1.7.12)

[error] property "save_gradle_cache" is not defined in object type {}

(expression)